test/e2e/load_balancer_type_test.go

```diff
@@ -0,0 +1,75 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadBalancerType(t *testing.T) {
+	t.Parallel()
+
+	t.Run("list", func(t *testing.T) {
+		t.Run("table", func(t *testing.T) {
+			out, err := runCommand(t, "load-balancer-type", "list")
+			require.NoError(t, err)
+			assert.Regexp(t,
+				NewRegex().Start().
+					SeparatedByWhitespace("ID", "NAME", "DESCRIPTION", "MAX SERVICES", "MAX CONNECTIONS", "MAX TARGETS").Newline().
+					AnyTimes(NewRegex().
+						Int().Whitespace().
+						Identifier().Whitespace().
+						AnyString().Whitespace().
+						Int().Whitespace().
+						Int().Whitespace().
+						Int().Newline()).
+					End(),
+				out,
+			)
+		})
+
+		t.Run("json", func(t *testing.T) {
+			out, err := runCommand(t, "load-balancer-type", "list", "-o=json")
+			require.NoError(t, err)
+			assert.True(t, json.Valid([]byte(out)), "is valid JSON")
```

Comment:
This isn't testing much, should we test the data inside?

Reply:
Load balancer type do not change every day, I am not against hardcoded values.

Or at least test the shape of the json document?

Reply:
Not necessarily, we could only check for the first element.